lldb/source/Commands/CommandObjectThread.cpp

```diff
@@ -970,19 +969,26 @@ class CommandObjectThreadUntil : public CommandObjectParsed {
           return;
         }
 
-        AddressRange fun_addr_range = sc.function->GetAddressRange();
-        Address fun_start_addr = fun_addr_range.GetBaseAddress();
-        line_table->FindLineEntryByAddress(fun_start_addr, function_start,
-                                           &index_ptr);
-
-        Address fun_end_addr(fun_start_addr.GetSection(),
-                             fun_start_addr.GetOffset() +
-                                 fun_addr_range.GetByteSize());
-
-        bool all_in_function = true;
+        uint32_t lowest_func_idx = UINT32_MAX;
```

Comment:
Would this be useful anywhere else?  It's asking a Function what its highest and lowest indexes are in the linetable from its CU.